torchdata/dataloader2/reading_service.py

```diff
@@ -540,9 +560,12 @@ def checkpoint(self) -> bytes:
         states = []
         for rs in self.reading_services:
             states.append(rs.checkpoint())
+        return b"\n".join(states)
```

Comment:
This is more of a placeholder for now? To be changed later?

Reply:
This is a valid solution to combine multiple `bytes`, unless you have any concern on this solution.